lib/livebook_web/live/output/plain_text_component.ex

```diff
@@ -42,7 +48,17 @@ defmodule LivebookWeb.Output.PlainTextComponent do
       phx-update="stream"
       phx-no-format
     ><span
-      :for={{dom_id, chunk}<- @streams.chunks} id={dom_id}>{chunk.text}</span></div>
+      :for={{dom_id, chunk}<- @streams.chunks} id={dom_id} style={format_style(chunk.style)}>{chunk.text}</span></div>
```

Comment:
@jonatanklosko what would be the best way to test this? Should we start adding unit tests to these components?

Reply:
We have a few integration tests like this:

https://github.com/livebook-dev/livebook/blob/af5f0de0509368360d5b599fbd3a7452b8c356e2/test/livebook_web/live/session_live_test.exs#L623-L635

Those are mostly for output updates though, because it involves the `send_update` and it's more likely to introduce a bug by changing an id or similar. I think to test the style, a component test would totally work, especially that now it accepts the whole `output`.

Reply:
Unit test pushed.